assembly/std/__tests__/arguments.spec.ts

```diff
@@ -29,6 +29,16 @@ describe('Args tests', () => {
     expect(args2.nextU64()).toBe(97);
   });
 
```

Comment:
is it a good idea to stay on aspect ?

Reply:
not at all but we have an issue for migrating all aspect tests